frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -989,12 +999,13 @@ async function getFirstButtonAndClick(page: Page, testID: string) {
 
 async function getCellTypeFmgButtonAndClick(page: Page, cellType: string) {
   await waitForElement(page, CELL_TYPE_LABELS_ID);
+
   await tryUntil(
     async () => {
       await page
-        .getByTestId(CELL_TYPE_LABEL_ROW_TEST_ID)
-        .filter({ hasText: new RegExp(`^${cellType}$`) })
-        .getByTestId(MARKER_GENE_BUTTON_TEST_ID)
+        .getByRole("img", {
```

Comment:
Playwright debugger suggests this locator and it works! Somehow the original locator doesn't work anymore